File: Python/bee_1035.py
a, b, c, d = map(int, input().split())
va = 0
if b > c and d > a and c + d > a + b:
    va = 1
if c > 0 and d > 0 and a % 2 == 0:
    va = va + 1
if va == 2:
    print('Valores aceitos')
else:
    print('Valores nao aceitos')

File: C/bee_1035.c
#include <stdlib.h>
#include <locale.h>
#include <math.h>
int main()
{
  int a, b, c, d, v;
  scanf("%d %d %d %d", &a, &b, &c, &d);
  v = 0;
  if (b > c & d > a & c + d > a + b){
      v = 1;
  }
  if (c > 0 % d > 0 & a % 2 == 0){
      v = v + 1;
  }

  if (v == 2){
    printf("Valores aceitos\n");
  }

  else{
     printf("Valores nao aceitos\n");
  }

  return(0);
}
